src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx

```diff
@@ -183,30 +184,34 @@ export class ConnectedFilterControl extends React.Component<
     }
     return actionsToReturn;
   }
-}
 
-function popoverFrom(actions: PopoverableAction[]): React.ReactElement<any>[] {
-  return actions.map((action) => {
+  private activatorButtonFrom(action: BaseAction): React.ReactElement<any> {
     return (
-      <Item key={action.key}>
-        <Popover
-          active={action.popoverOpen}
-          activator={activatorButtonFrom(action)}
-          onClose={action.onAction}
-          preferredAlignment="left"
-          sectioned
-        >
-          {action.popoverContent}
-        </Popover>
-      </Item>
+      <Button
+        onClick={action.onAction}
+        disclosure
+        disabled={this.props.disabled}
+      >
+        {action.content}
+      </Button>
     );
-  });
-}
+  }
 
-function activatorButtonFrom(action: BaseAction): React.ReactElement<any> {
-  return (
-    <Button onClick={action.onAction} disclosure>
-      {action.content}
-    </Button>
-  );
+  private popoverFrom(actions: PopoverableAction[]): React.ReactElement<any>[] {
+    return actions.map((action) => {
+      return (
+        <Item key={action.key}>
+          <Popover
+            active={this.props.disabled ? false : action.popoverOpen}
```

Comment:
```suggestion
            active={action.popoverOpen}
```

I don't think this needs to be conditional since the popover activator is `disabled` when the filter is `disabled`.